app/components/about/Testimonials.tsx

```
@@ -0,0 +1,245 @@
+                                        <div className="w-10 h-10 bg-gray-200 rounded-full"></div> {/* Reduced from w-12 h-12 to w-10 h-10 */}
+                                        <div className="ml-3"> {/* Reduced from ml-4 to ml-3 */}
+                                            <h3 className="font-semibold">
+                                                {testimonials[testimonials.length - 1].name}
+                                            </h3>
+                                            <p className="text-gray-600 text-sm">
+                                                {testimonials[testimonials.length - 1].affiliation}
+                                            </p>
+                                        </div>
+                                    </div>
+                                    <p className="text-gray-700">
+                                        "{testimonials[testimonials.length - 1].quote}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)